bringup/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+# ###############################################################################
+# Set minimum required version of cmake, project name and compile options
+# ###############################################################################
+cmake_minimum_required(VERSION 3.5)
```

```suggestion
cmake_minimum_required(VERSION 3.22)
```
Ubuntu 22 ships with CMake 3.22 so you can require that. This opts you into better, newer default behaviors.